src/grt/grt-wave_opt_file-parse-debug.adb

```diff
@@ -39,22 +42,24 @@ package body Grt.Wave_Opt_File.Parse.Debug is
          else
             Put_Line ("Instances : ");
          end if;
-         Dump_Sub_Tree (Trees (Index), 1);
+         Dump_Sub_Tree (Trees (Index));
       end loop;
       Put_Line ("----------- END -----------------");
       New_Line;
    end Dump_Tree;
 
-   procedure Dump_Sub_Tree (Cursor : Elem_Acc; Level : Positive)
+-------------------------------------------------------------------------------
```

Comment:
Ok. That was a way to separate the body of the subprogramms defined in 
the public part of a package specification from subprogramms local to 
the package body.

I will fix that if you prefer.


Le 27. 07. 16 à 19:23, tgingold a écrit :
>
> In src/grt/grt-wave_opt_file-parse-debug.adb 
> <https://github.com/tgingold/ghdl/pull/124#discussion_r72483207>:
>
> >        end loop;
> >        Put_Line ("----------- END -----------------");
> >        New_Line;
> >     end Dump_Tree;
> >
> > -   procedure Dump_Sub_Tree (Cursor : Elem_Acc; Level : Positive)
> > +-------------------------------------------------------------------------------
>
> No need to fix that immediately, but I am not a fan of these caret lines.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/tgingold/ghdl/pull/124/files/eae288207d481e422e395aa7d310fa629d20432b#r72483207>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ATNePjDpnuJYa0isqcTerAkr2dhhpETtks5qZ5QJgaJpZM4JVgTD>.
>